torch/ao/sparsity/experimental/activation_sparsifier/activation_sparsifier.py

```diff
@@ -95,24 +99,104 @@ def __init__(self, model: nn.Module, aggregate_fn=None, reduce_fn=None, mask_fn=
 
         self.state: Dict[str, Any] = defaultdict(dict)  # layer name -> mask
 
+    @staticmethod
+    def __safe_rail_checks(args):
```

Comment:
No "private" classes -- we discourage the use of name mangling. Please, use a single underscore